dwave/system/samplers/dwave_sampler.py

```diff
@@ -89,9 +89,17 @@ class DWaveSampler(dimod.Sampler, dimod.Structured):
 
     Uses parameters set in a configuration file, as environment variables, or
     explicitly as input arguments for selecting and communicating with a D-Wave
-    system. For more information, see
+    system, as described in
     `D-Wave Cloud Client <https://docs.ocean.dwavesys.com/en/stable/docs_cloud/sdk_index.html>`_.
 
+    :term:`Solver` selection---handled by :ref:`dwave-cloud-client <sdk_index_cloud>`'s
+    :meth:`~dwave.cloud.client.Client.get_solvers` method---defaults to filtering
+    solvers you have access to by
+    `solver properties <https://docs.dwavesys.com/docs/latest/c_solver_3.html>`_
+    ``category=qpu`` and ``supported_problem_type=qubo/ising`` and returns online
+    solvers ordered by highest number of qubits.
```

Comment:
May be true effectively, but it would be nice to be technically more correct: 
- solver features are _not_ constrained at all. We filter by `client=qpu`. Which, down the stack uses category, but the point is `solver` is undefined in the library code, so it can be set in the user config file/env/code.
- `order_by` is only hinted, so it's overridden if user defines `solver` on any level
- supported problem type is not constrained

Reply:
Let's discuss this. My intention was to give the links to dwave-cloud-client for users who want to dive into the full complexities of solver selection at all its levels and here to describe just the basic default. I'm not sure we want such level of cloud-client detail in dwave-system, especially for the hybrid samplers.

Reply:
I was trying to say we should _remove_ some details from here. Like `supported_problem_types` is not checked and `solver` does not overwrite user-supplied definition in either config file or env.

Reply:
Yes, I'm making such changes now